ui/components/app/dropdowns/network-dropdown.js

```diff
@@ -174,7 +174,7 @@ class NetworkDropdown extends Component {
             <div className="network-check__transparent">✓</div>
           )}
           <ColorIndicator
-            color={opts.isLocalHost ? 'localhost' : COLORS.TEXT_MUTED}
+            color={opts.isLocalHost ? 'localhost' : COLORS.ICON_MUTED}
             size={SIZES.LG}
             type={ColorIndicator.TYPES.FILLED}
           />
```

Comment:
The network indicators for custom networks uses multiple grays this is an attempt to consolidate them

## Before
<img width="938" alt="before" src="https://user-images.githubusercontent.com/8112138/173706541-4afdfdfe-418f-4a32-96d5-7c26fb133947.png">

## After
<img width="977" alt="Screen Shot 2022-06-14 at 4 25 56 PM" src="https://user-images.githubusercontent.com/8112138/173706563-35fb983d-6154-4696-9ce0-832938b60d05.png">